### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by rtertiaer**
I'd like to implement a pull request template to set expectations and bowling bumpers around feature/bug development. This will particularly help us accept quality contributions from new contributors.

Did I miss anything? Does this feel extra? I borrowed most of this from [here](https://github.com/stevemao/github-issue-templates/blob/master/checklist/PULL_REQUEST_TEMPLATE.md), which has a host of other pull request templates to peruse.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# What does this change intend to accomplish?
+
+ 
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
```
Comment (linknum23) [inline .github/pull_request_template.md:6]: We don't have this yet!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### New Feature Submissions:
+
```
Comment (linknum23) [inline .github/pull_request_template.md:12]: NOTE:  ./scripts/test does both of these things!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] Does your submission pass tests?
+* [ ] Have you lint your code locally prior to submission?
+
+### Changes to Core Features:
+
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you written new tests for your core changes, as applicable?
+* [ ] Have you successfully ran tests with your changes locally?
+
```
Comment (linknum23) [inline .github/pull_request_template.md:21]: Have you updated the documentation/user manual, as applicable?

Comment (Lohrer) [reply .github/pull_request_template.md:21]: Oooo I like this one. Also, have you updated the changelog, as applicable.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# What does this change intend to accomplish?
+
+ 
+### All Submissions:
```
Comment (Lohrer) [inline .github/pull_request_template.md:4]: I'm not really sure we need to break things up into All/New Features/Core Changes categories, seems like all of these checks apply to any changes. Also, tests passing is listed under both New Features/Core Changes.

Comment (rtertiaer) [reply .github/pull_request_template.md]: totally. it's pretty condensed in the latest rendition; check it out :+1:

File: .github/pull_request_template.md (hunk 3 above)
Comment (rtertiaer) [reply .github/pull_request_template.md]: implemented~

File: .github/pull_request_template.md (hunk 2 above)
Comment (rtertiaer) [reply .github/pull_request_template.md]: I made a note in the testing line to try `scripts/test`; it'll also help us keep that script updated going forward :)

File: .github/pull_request_template.md (hunk 1 above)
Comment (rtertiaer) [reply .github/pull_request_template.md]: removed~

**Review by Lohrer — APPROVED** [review]
Seems like a good initial version of this. After getting a few ~miles~ PRs on this we should double-check it turned out to make sense and there isn't anything else to add.

**Pull request merged**